.github/workflows/triage_add_to_routed_project.yml

```diff
@@ -0,0 +1,39 @@
+name: Triage add to routed-to project
```

Comment:
I'm not sure what this name is supposed to mean 😬 is it supposed to be something like "Add to routed-to column in Triage Project"?

Reply:
I guess, could it be `Triage: route to team project board`?

Reply:
since this is moving to the correct team-focused project board, not the triage board

Reply:
updated naming: [f0f4e68](https://github.com/cypress-io/cypress/pull/23613/commits/f0f4e68f58f82ebed7425418812c89f3abf1eea6)